src/NRRD.jl

```diff
@@ -742,51 +742,63 @@ function get_axes(header, nd)
     else
         sd = get(header, "space dimension", nd)
     end
+
     for f in per_spacedim
         if haskey(header, f)
             length(header[f]) == sd || error("expected $sd dimensions in field $f, got $(header[f])")
         end
     end
+
     ## Test whether we have fields that require an AxisArray
     axes_fields = setdiff(per_axis, ["sizes", "kinds", "centers", "centerings"])
     need_axes = haskey(header, "space")
     for f in axes_fields
         need_axes |= haskey(header, f)
     end
+
     # also check for a time axis
-    istime = falses(nd)
-    isspace = falses(nd)
+    is_time = falses(nd)
```

Comment:
Hi, thank you for the contribution! Can you please separate out the style/formatting changes from functional changes?